packages/jest-haste-map/src/index.js

```diff
@@ -234,6 +231,8 @@ class HasteMap extends EventEmitter {
     this._workerPromise = null;
     this._workerFarm = null;
     this._watchers = [];
+    this._moduleIDsByFilePath = new Map();
+    this._moduleArraysByIDAndPlatform = new Map();
```

Comment:
If I'm being super annoying I'd ask you to sort these alphabetically within the block.